theano/tensor/basic.py

```diff
@@ -3179,11 +3179,16 @@ def var(input, axis=None, ddof=0, keepdims=False):
         If this is set to True, the axes which are reduced are
         left in the result as dimensions with size one. With this option,
         the result will broadcast correctly against the original tensor.
+    corrected : bool
+        If this is set to True, the 'corrected_two_pass' algorithm is
+        used to compute the variance.
+        Refer : http://www.cs.yale.edu/publications/techreports/tr222.pdf
 
     Notes
     -----
-    It uses the two-pass algorithm for more stable results.
+    By default, uses the two-pass algorithm for more stable results.
```

Comment:
> By default, uses the two-pass algorithm

This refers to the usual two-pass algorithm used default for calculating the variance, not the corrected version. I will rewrite to make it more clear.